datumaro/plugins/datumaro_format/converter.py

```diff
@@ -143,11 +146,13 @@ def add_item(self, item: DatasetItem):
                 converted_ann = self._convert_caption_object(ann)
             elif isinstance(ann, Cuboid3d):
                 converted_ann = self._convert_cuboid_3d_object(ann)
+            elif isinstance(ann, Skeleton):
+                converted_ann = self._convert_skeleton_object(ann)
             else:
                 raise NotImplementedError()
             annotations.append(converted_ann)
 
-    def add_categories(self, categories):
+    def add_categories(self, categories: dict[AnnotationType, Categories]):
```

Comment:
There is `dataset.components.extractor.CategoriesInfo` for this.

Reply:
fixed